documentation/modules/post/linux/gather/haserl_read.md

```diff
@@ -0,0 +1,59 @@
+## Vulnerable Application
+
+[Haserl](http://haserl.sourceforge.net/) is an unmaintained tool to use LUA as CGI in web servers.
+On Linux, if haserl is usually suid root, sot that it will attempt to drop its privilege to the uid/gid of the owner of the cgi script,
+similar to suexec in Apache.
```

Comment:
This sentence does not run well. Perhaps something like this:
```suggestion
On Linux, when haserl is suid root, it will attempt to drop its privilege to the uid/gid of the owner of the cgi script,
similar to suexec in Apache.
```